### PR TITLE
Fix issue with ELB when Weave Cloud token is set (close #102)

### LINKED ISSUE #102: having specified to Weave Cloud service token breaks ELB created by CFN
Because we configure the ELB to forward application port (80) and the Scope app port (4040), it decides that all instances are unhealthy when Scope app is missing.

### PR DESCRIPTION
**Pull request by errordeveloper**
This also updates references to Weave Cloud

### CODE REVIEW

File: aws/ecs/cloudformation-identiorca.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "Description": "Weave-enabled AWS CloudFormation template to create resources required to run tasks on an ECS cluster.  This template uses a larger example app.",
   "Outputs": {
     "AppFrontend": {
+      "Condition": "DoDeployExampleApp",
```
Comment (2opremio) [inline aws/ecs/cloudformation-identiorca.json]: Where is this condition initialized? (sorry if it's a stupid question, I am not too familiar with CF)

File: aws/ecs/cloudformation-identiorca.json [hunk 2]
```diff
@@ -155,16 +157,12 @@
         "Yes"
       ]
     },
-    "UseScopeService": {
-      "Fn::Not": [
+    "UseWeaveScopeStandalone": {
```
Comment (2opremio) [inline aws/ecs/cloudformation-identiorca.json]: Wouldn't it be simpler to re-use the WeaveCloudServiceToken and check if it's empty in the ELB listener section?

File: aws/ecs/cloudformation-identiorca.json (hunk 1 above)
Comment (errordeveloper) [inline aws/ecs/cloudformation-identiorca.json]: It's down below, in `Conditions` section.

File: aws/ecs/cloudformation-identiorca.json (hunk 2 above)
Comment (errordeveloper) [inline aws/ecs/cloudformation-identiorca.json]: Unfortunately it's not that simple, we need to have a conditioned defined, and then we can use `"Condition": "UseWeaveScopeStandalone"` or `"Fn::If": [ "UseWeaveScopeStandalone", <expr>, <else_expr> ]`. In `Fn::If` blocks we can check `WeaveCloudServiceToken`, but it'd make those longer and cause more duplication, and we cannot do the same for  `Condition` parameter. We have `"Condition": "UseWeaveScopeStandalone"` in `Outputs` section, for example.

Comment (2opremio) [inline aws/ecs/cloudformation-identiorca.json]: Oh, so if blocks don't support inline boolean expressions?

File: aws/ecs/cloudformation-identiorca.json (hunk 1 above)
Comment (2opremio) [inline aws/ecs/cloudformation-identiorca.json]: Not part of this patch (sorry, I didn't see it)

File: aws/ecs/cloudformation-identiorca.json (hunk 2 above)
Comment (errordeveloper) [inline aws/ecs/cloudformation-identiorca.json]: That's right, but my explanation above does say that and it is wrong.
Here is what [the docs](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-conditions.html#d0e118119) say:
```
"Fn::If": [condition_name, value_if_true, value_if_false]
```

Comment (2opremio) [inline aws/ecs/cloudformation-identiorca.json]: Fair enough, thanks for the explanation.
